packages/firebase/useAuth/index.ts

```diff
@@ -1,5 +1,6 @@
 import { computed, ComputedRef, ref, Ref } from 'vue-demi'
-import firebase from 'firebase'
+import firebase from 'firebase/app'
+import 'firebase/auth'
```

Comment:
This introduced a side effect to this package which I would like to avoid. Is this possible to be done on user side?

Reply:
Do you mean the 'import 'firebase/auth' line? If so then we can probably remove that and just keep `import firebase from 'firebase/app` line? If they are using the `useAuth` function then they will end up having to import `firebase/auth` anyway, so we should be able to safely remove it. Does that work for you?

Reply:
Yep, thanks!